internal/nodes/ingress_model_rel.go

```diff
@@ -311,6 +311,9 @@ func IngressChanges(ingName string, namespace string, key string) ([]string, boo
 
 		if ingObj.Spec.IngressClassName != nil {
 			objects.SharedSvcLister().IngressMappings(metav1.NamespaceAll).UpdateIngressClassMappings(namespace+"/"+ingName, *ingObj.Spec.IngressClassName)
+		} else if aviIngClassName, found := isAviLBDefaultIngressClass(); found {
```

Comment:
I wonder what would happen if the ingress class is added after adding the ingress. Also take a look at this: 
https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/299
There might be come conflicts.